src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,87 @@
+import classNames from 'classnames';
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types/Person';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+const getParent = (
+  array: Person[], parentName: string | null,
+): Person | null => {
+  return array.find(parent => parent.name === parentName) || null;
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { personId } = useParams();
```

Comment:
You should use the correct type for useParams hook. Consider defining a type for the route parameters like `type RouteParams = { personId: string };` and then use it with useParams: `const { personId } = useParams<RouteParams>();`.